pandas/tests/arithmetic/test_numeric.py

```diff
@@ -0,0 +1,534 @@
+# -*- coding: utf-8 -*-
+# Arithmetc tests for DataFrame/Series/Index/Array classes that should
+# behave identically.
+# Specifically for numeric dtypes
+from datetime import timedelta
+from decimal import Decimal
+
+import pytest
+import numpy as np
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas.compat import PY3
+from pandas import Timedelta, Series, Index, TimedeltaIndex
+
+
+@pytest.fixture(params=[pd.Float64Index(np.arange(5, dtype='float64')),
+                        pd.UInt64Index(np.arange(5, dtype='uint64')),
+                        pd.Int64Index(np.arange(5, dtype='int64')),
+                        pd.RangeIndex(5)],
+                ids=lambda x: type(x).__name__)
+def idx(request):
```

Comment:
duplicate of conftest (maybe that's ok)

Reply:
Yah, I'm not going to worry about fixture duplication until after tests.test_arithmetic is moved to tests.arithmetic.test_timedelta64